library/src/morpheus-master/sass/modules/tool/profile2/_profile2.scss

```diff
@@ -60,8 +63,11 @@
 		clip: rect(1px, 1px, 1px, 1px);
 	}
 
-	#myPhoto:hover > .edit-image-button, .edit-image-button:focus {
+	.btn-holder:hover > .edit-image-button,
+	.edit-image-button:focus > .edit-image-button:focus-visible,
+	.edit-image-button:focus-visible {
```

Comment:
`:focus-visible` is usually used to show the blue outline over input elements when keyboard is used, and not when using mouse.

In this case is specially useful, as `edit-image-button` in meant to show only on hover, so when the Edit modal is closed, focus gets back to this button through JS and thanks to SCSS is 'converted' to `:focus-visible`.

Therefore, if you're only using mouse, and the cursor isn't hovering the img after you **close the Editor modal**, the button will hide. However, it will remain if you're interacting with keyboard, in order to get to the last used element without needing to hit the tab key X times.

Best behaviour for each scenario : )